schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/spi/ActivityCreator.java

```diff
@@ -106,8 +134,8 @@ private Activity createActivity(Activity from, ActivityType activityType, Repeat
             if (repeatDays.contains(start.getDayOfWeek())) {
                 break;
             }
-            start = start.plusDays(1);
-            finish = finish.plusDays(1);
+            start = increment(start, 1);
+            finish = increment(finish, 1);
```

Comment:
A duration approach does not work if the start is before daylight savings and the finish is after daylight savings.